.github/workflows/long-running-azure.yaml

```
@@ -436,8 +451,8 @@ jobs:
             exit 1
           fi
 
-          # Poll logs for up to  iterations, 30 seconds each (upto 3 minutes total)
-          for i in {1..6}; do
+          # Poll logs for up to 20 iterations, 30 seconds each (up to 10 minutes total)
+          for i in {1..20}; do
```

Why don't we just retry this step on failure instead of adding retry logic to the register manifest flow? We use something like this in samples repo: https://github.com/radius-project/samples/blob/edge/.github/workflows/test.yaml#L397.

---

I think we can move to this later once we see it stabilized. The current approach gives me the exact error of the logs  which is valuable right now.